README.md

```diff
@@ -202,6 +202,10 @@ Defaults to `default`
 
 Maximum amount of time in minutes YUM randomly waits before running. Valid values are numbers between `0` and `1440`. `0` to disable. Defaults to `60`
 
+#####`systemname`
+
+Hostname to use for identification. Defaults to '' which should be automatically converted to the system hostname.
```

Comment:
Since `undef` is a better default (see below) you can replace the second sentence with *"Defaults to the system hostname if undefined"*.